app/main/posts/detail/post-value.directive.js

```diff
@@ -18,6 +18,21 @@ module.exports = ['PostEndpoint', function (PostEndpoint) {
                     return PostEndpoint.get({ id : entry });
                 });
             }
+            if ($scope.attribute.type === 'datetime' && $scope.attribute.input === 'date') {
```

Comment:
Could this be a single outer if($scope.attribute.type==='datetime') 
And then split out ifs beneath it, I'm not sure if that would be clearer